Testing/functional/tests/cypress/integration/e2e/covid19/authenticatedVaccineCard.js

```diff
@@ -7,7 +7,17 @@ describe("Authenticated Vaccine Card", () => {
             "getVaccinationStatus"
         );
 
-        cy.enableModules(["Immunization"]);
+        cy.configureSettings(
+            {
+                covid19: {
+                    proofOfVaccination: {
+                        exportPdf: true,
+                    },
+                },
+            },
+            ["Immunization"]
+        );
```

Comment:
I don't think exportPdf needs to be set in this test or many of the others, only in ones that used to enable "VaccinationExportPdf".

Reply:
No it has to be there.